src/Units/MoveTypes/MoveType.java

```diff
@@ -88,6 +88,8 @@ public boolean canTraverse(Environment tile)
   /** Sets the cost to move through terrain during weather. */
   public void setMoveCost(Weathers weather, TerrainType terrain, int cost)
   {
+    if( cost > IMPASSABLE )
+      cost = IMPASSABLE;
```

Comment:
Does this happen?

Reply:
I originally added +1 to all movement costs in rain, which was awful.
So yes, that would happen. I didn't find anything wrong, but I don't want to run into a subtle bug down the line.